README.md

```diff
@@ -1,34 +1,56 @@
-# MAPI Inspector for Fiddler
-The Messaging Application Programming Interface (MAPI) Inspector for [Fiddler](http://www.telerik.com/fiddler) decodes the MAPI message payload of an HTTP POST request and response. The MAPI Inspector displays under the *Inspectors* tab in Fiddler.

```

Comment:
Thanks Clare for helping on this. Great content. One thinking is do you think we should give separate documentation for MAPI and Fiddler separately? Meaning give a TOC like thing for the readme.md in the root path, and link to each readme under MAPI and Fiddler. And in that readme inside MAPI and Fiddler folder, talk about each only? Reason is i'm thinking customer may only care about one category of parsers, mixing them together may give them extra info they want.